sdk/simulator-service/src/implementation/strategies/refinance.ts

```diff
@@ -0,0 +1,179 @@
+import { FlashloanProvider, Simulation, SimulationSteps } from '@summerfi/sdk-common/simulation'
+import { getReferencedValue, makeStrategy } from '~swap-service/implementation/helpers'
+import { LendingPool } from '@summerfi/sdk-common/protocols'
+import { Simulator } from '~swap-service/implementation/simulator-engine'
+import { Token, TokenAmount, Position } from '@summerfi/sdk-common/common/implementation'
+import { SimulationType } from '@summerfi/sdk-common/orders'
+import { newEmptyPositionFromPool } from '@summerfi/sdk-common/common/utils'
+
+export const refinanceStrategy = makeStrategy([
+    {
+        step: SimulationSteps.Flashloan,
+        optional: false,
+    },
+    {
+        step: SimulationSteps.PaybackWithdraw,
+        optional: false,
+    },
+    {
+        step: SimulationSteps.Swap,
+        optional: true,
+    },
+    {
+        step: SimulationSteps.DepositBorrow,
+        optional: false,
+    },
+    {
+        step: SimulationSteps.Swap,
+        optional: true,
+    },
+    {
+        step: SimulationSteps.PaybackFlashloan,
+        optional: false,
+    },
+    {
+        // In case of target debt being different then source debt we need a swap,
+        // We cannot forsee the exact amount of the swap, so we need to return excess tokens to user
+        step: SimulationSteps.ReturnFunds,
+        optional: true,
+    },
+])
+
+interface Quote {
```

Comment:
Is this interface exclusive to refinance? If not, please add a TODO to indicate that we may need to extract it in the future so it can be reused by other strategies (no need to extract it now)